docs/tutorials/07_leveraging_qiskit_runtime.ipynb

```diff
@@ -7,6 +7,13 @@
     "# Leveraging Qiskit Runtime "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**WARNING:** The `VQEClient` discussed in this tutorial is **DEPRECATED** as of version 0.6 of Qiskit Nature and will be removed no sooner than 3 months after the release. Instead, users should migrate their code to use the Qiskit     Runtime Primitives. A guide on how to use this can be found [here](https://qisk.it/algo_migration#vqe)."
```

Comment:
Could we use a Sphinx ..warning :: here instead. This will produce a more noticeable highlighted box in the resultant html we publish - yes the notebook source will show the tag but unless you get/run the notebook you will not see that and I don't think its all that bad even if one does!

> Instead, users should migrate their code to use the Qiskit     Runtime Primitives.

Can we also use the more VQE specific text as it was updated elsewhere for this

Reply:
I was wondering if this is possible. I didn't think it was since this is a jupyter notebook...
I will try again locally and update